docs-website/docs/snippets/spark-connectors/checkpoint-dir.md

```diff
@@ -0,0 +1,3 @@
+:::tip Streaming Data Platform
```

Comment:
@claudiofahey I needed these files for everything to render, but I guess they could be empty here?

Reply:
Yes.